NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # renv 0.18.0  (UNRELEASED)
 
+* `status()` gets new, more compact, display when packages have some 
```

Comment:
```suggestion
* `renv::status()` gets new, more compact, display when packages have some 
```

I try to stick to using `renv::` as a prefix because we often encourage users to prefix usages of `renv` in a similar way; e.g. to avoid confusion about `base::load()` versus `renv::load()`.